apps/meteor/app/livechat/imports/server/rest/users.ts

```diff
@@ -9,16 +9,15 @@ import { hasAtLeastOnePermissionAsync } from '../../../../authorization/server/f
 import { findAgents, findManagers } from '../../../server/api/lib/users';
 import { Livechat } from '../../../server/lib/Livechat';
 
+const emptyStringArray: string[] = [];
+
 API.v1.addRoute(
 	'livechat/users/:type',
 	{
 		authRequired: true,
 		permissionsRequired: {
-			GET: {
-				permissions: ['manage-livechat-agents'],
-				operation: 'hasAll',
-			},
-			POST: { permissions: ['view-livechat-manager'], operation: 'hasAll' },
+			'POST': ['view-livechat-manager'],
+			'*': emptyStringArray,
```

Comment:
```suggestion
			'GET': [],
```

Wouldn't keeping it explicit be better? just-in-case we have future PUT or DELETE endpoints on this path

Reply:
This is the explicit way. Having by method forces you to define "by method". The * here means "No permission _unless_ specified otherwise"